packages/logger-plugin/src/logger.plugin.ts

```diff
@@ -18,7 +16,7 @@ export class NgxsLoggerPlugin implements NgxsPlugin {
   ) {}
 
   handle(state: any, event: any, next: NgxsNextPluginFn) {
-    if (this._options.disabled) {
+    if (this._options.disabled || !this._options.filter!(event, state)) {
```

Comment:
`this._options.filter!` 
- I think you need to check for null

Reply:
But it cannot be `null`. The `loggerOptionsFactory` is passing in a default predicate.